packages/dotcom-ui-header/src/enhanced-search/enhancedSearch.js

```diff
@@ -12,6 +12,8 @@ class EnhancedSearch extends TopicSearch {
           searchTerm: this.searchEl.value,
           suggestions: {}
         })
+        const detail = { category: 'search', action: `Error: ${error.message}` }
```

Comment:
Let's save this for a separate PR I think, the contents of this payload are pretty standardised, e.g. i think we have to choose from a standard set of values for `category` and `action`!

It might take some time to figure this out, so let's not bog down the other fixes while we figure this one out.